apiserver/plane/app/views/view/base.py

```
@@ -358,12 +370,14 @@ def listCustomProperty(self, request, slug):
                 "total_results": 0,
                 "limit": 10,
                 "total_pages": 1,
-                "page": 1
+                "page": 1,
+                "data_type": None
             }
         
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)